grammar.js

```diff
@@ -78,6 +78,11 @@ module.exports = grammar({
     [$._type, $._pointer_base_type],
     [$._type, $._ref_base_type],
 
+    [$._type_name, $._nullable_base_type],
+    [$._type_name, $._array_base_type],
+    [$._type_name, $._pointer_base_type],
+    [$._type_name, $._ref_base_type],
+
```

Comment:
Can you rebase and run with the line number scripts. I'm getting anxious about all these conflict rules.

Reply:
After the rebase, these are not needed. I've pushed a commit with the file sizes, there's basically no change.